Resources/Prototypes/Recipes/Construction/Graphs/weapons/metal_hydrogen_axe.yml

```
@@ -0,0 +1,14 @@
+﻿# Assmos - Metal Hydrogen
+- type: constructionGraph
+  id: MetalHydrogenAxeConstruct
+  start: start
+  graph:
+    - node: start
+      edges:
+      - to: icon
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)